### DISCUSSION TITLE
Can't change color of formatSpecifier

### DISCUSSION THREAD

**Discussion by sprtlw** [mrcjkb/rustaceanvim#479] [Q&A]
On https://rust-analyzer.github.io/manual.html#semantic-syntax-highlighting it shows i can change formatSpecifier to make curly brackets {} stand out from a string when using format macros and the like. I attempted to override it with the following in my chadrc file.
```
M.ui = {
  theme = 'chocolate',
  hl_override = {
    ["@lsp.type.attributeBracket"] = { fg = "#d790f8" },
  },
}
```
This does not change the bracket color, it still shows as the string color, which is weird considering \n properly shows as a different color. Also, when running :echo synIDattr(synIDtrans(synID(line('.'), col('.'), 1)), 'name') the curly brackets are being detected as part of the string, not attributeBracket.
![image](https://github.com/user-attachments/assets/5b220d01-0633-441e-90f7-6964041bbc85)

Any help as to get this working would be appreciated.

**Comment by mrcjkb** [comment]
Hey 👋 

You can use `:Inspect` to see the highlights that are being applied.
